src/Tribe/Values/Abstract_Currency.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tribe\Values;
+
+abstract class Abstract_Currency extends Abstract_Value implements Currency_Interface {
```

Comment:
Docblocks for all methods. Set the Docblocks here, and `@inheritDoc` on all others